dynamic_rest/viewsets.py

```diff
@@ -340,6 +340,23 @@ class DynamicModelViewSet(WithDynamicViewSetMixin, viewsets.ModelViewSet):
     ENABLE_BULK_PARTIAL_CREATION = settings.ENABLE_BULK_PARTIAL_CREATION
     ENABLE_BULK_UPDATE = settings.ENABLE_BULK_UPDATE
 
+    def is_bulk_request(self, request):
+        plural_name = self.get_serializer_class().get_plural_name()
+        if isinstance(request.data, list):
+            return True
+        elif plural_name in request.data and len(request.data) == 1:
+            return True
+        return False
+
+    def _get_bulk_payload(self, request):
```

Comment:
This and `is_bulk_request` look nearly identical. I think it'd be DRYer if this method just returned `None` if it's not a bulk request, and the code in update/delete was basically:

```python
bulk_payload = self._get_bulk_payload(request)
if bulk_payload:
    # do bulk operation
```